core/tests/protractor_utils/AdminPage.js

```diff
@@ -266,6 +276,60 @@ var AdminPage = function() {
         });
       });
   };
+  var _assignReviewer = function(
+      username, reviewCategory, languageDescription = null) {
+    waitFor.elementToBeClickable(
+      adminRolesTab, 'Admin Roles tab is not clickable');
+    adminRolesTab.click();
+
+    waitFor.visibilityOf(
+      assignReviewerForm, 'Assign reviewer form is not visible');
+
+    var usernameInputField = assignReviewerForm.element(reviewerUsernameCss);
+    waitFor.visibilityOf(
+      usernameInputField,
+      'Username input field is not visible in assign reviewer form');
+    usernameInputField.sendKeys(username);
+
+    var reviewCategorySelectField = assignReviewerForm.element(
+      reviewCategorySelectCss);
+    waitFor.visibilityOf(
+      reviewCategorySelectField,
+      'Review category options are not visible in assign reviewer form');
+    var reviewCategoryOption = reviewCategorySelectField.element(
+      by.cssContainingText('option', reviewCategory));
+    reviewCategoryOption.click();
+
+    if (languageDescription !== null) {
+      var languageSelectField = assignReviewerForm.element(languageSelectCss);
+      waitFor.visibilityOf(
+        languageSelectField,
+        'Language options are not visible in assign reviewer form');
+      var languageOption = languageSelectField.element(
+        by.cssContainingText('option', languageDescription));
+      languageOption.click();
+    }
+
+    var submitButton = assignReviewerForm.element(reviewerFormSubmitButtonCss);
+    waitFor.elementToBeClickable(
+      submitButton, 'Submit assign reviewer button is not clickable');
+    submitButton.click();
+
+    waitFor.textToBePresentInElement(
+      statusMessage, 'Successfully added',
+      'Could not add translation reviewer successfully');
+  };
+  this.assignTranslationReviewer = function(languageDescription, username) {
+    _assignReviewer(username, 'TRANSLATION', languageDescription);
+  };
+
+  this.assignVoiceoverReviewer = function(languageDescription, username) {
+    _assignReviewer(username, 'VOICEOVER', languageDescription);
+  };
```

Comment:
Is there a reason to flip the argument order from the public to the private function (e.g. language then username in `assignTranslationReviewer` but username then language in `_assignReviewer`)? If not, this swap seems to add unnecessary complexity

Reply:
Done!